praxis-notes-app/src/fs/providers/servers/get-azure-container-client.provider.ts

```
@@ -0,0 +1,28 @@
+import { Function } from '@errors/types';
```

Biome (1.9.4)

[error] Do not shadow the global "Function" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)